src/pheval/post_processing/post_processing.py

```diff
@@ -450,7 +451,7 @@ def generate_pheval_result(
         ValueError: If the results are not all the same type or an error occurs during file writing.
     """
     if not pheval_result:
-        print(f"Warning: No results found for {tool_result_path.name}")
+        info_log.warning(f"No results found for {tool_result_path.name}")
```

Comment:
I wasn’t sure. I can change it to log an error but I didn’t to begin with because sometimes the tool doesn’t generate results and there isn’t an error with the tool itself

Reply:
Fair enough.